src/Network/HTTPException/LenghtRequiredException.php

```diff
@@ -23,7 +23,7 @@
 
 use Friendica\Network\HTTPException;
 
-class LenghtRequiredException extends HTTPException
+class LengthRequiredException extends HTTPException
```

Comment:
How come this isn't used anywhere?

Reply:
This is part of the mapping of HTTP error codes to PHP exceptions. We just happen not to be using this particular one for our purpose.